File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyListViewElementsRow.cpp

#include "SelectRowCommand_ViewMyListViewElementsRow.h"
#include "SelectRowCommand_ViewMyListViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewMyListViewElementsRow::getHeaderLabelText( ) 
  {
    return this->HeaderLabelText;
  }
  int SelectRowCommand_ViewMyListViewElementsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyListViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewMyListViewElementsRow.cs
/// </filename>


namespace commands
{
    internal class SelectRowCommand_ViewMyListViewElementsRow
  {
        private string HeaderLabelText;
     
        public     virtual string getHeaderLabelText()
    {
      return this.HeaderLabelText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

